command/certificate/create.go

```diff
@@ -263,21 +264,32 @@ $ step certificate create --template root.tpl \
 '''
 
 Create an intermediate certificate using the previous root. This intermediate
-will be able to sign also new intermediate certificates:
```

Comment:
By extending the 'maxPathLen` we are enabling this intermediate sign leaf and intermediate certificates.

Reply:
Fixed with 57b8a06